.github/workflows/release-build.yml

```
@@ -0,0 +1,201 @@
+          name: app-lts-release-aab
+          path: commcare-android/app/build/outputs/bundle/ltsRelease/app-lts-release.aab
+
+  browserstack-tests:
+    needs: build-test-assemble
+    uses: ./.github/workflows/browserstack-tests.yml
+    secrets:
+      HQ_API_PASSWORD: ${{ secrets.HQ_API_PASSWORD }}
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_PASSWORD: ${{ secrets.BROWSERSTACK_PASSWORD }}
+
+  release:
```

actionlint (1.7.11)

[error] job "release" needs job "build" which does not exist in this workflow

(job-needs)